packages/core/src/components/Icon/icons/LogOut.tsx

```diff
@@ -0,0 +1,21 @@
+import { SVGProps } from '@components/Icon/types';
+
+export const LogOut = ({ fill = '#000', size = 24 }: SVGProps) => (
+  <svg
+    width={`${size}px`}
+    height={`${size}px`}
+    fill="none"
+    viewBox="0 0 16 15"
```

Comment:
I think, we also need synchronize the viewBox, as width and height

Reply:
I think the viewbox should be left static because passing a size variable does not stretch the current icon to the entire svg container.
![Screenshot_1](https://github.com/ssagroup/ui-kit/assets/141251682/42fa61d3-818c-4c79-9ce8-2cc8b5680ef6)